internal/git_config/git_config.go

```diff
@@ -134,3 +152,50 @@ func SetOption(logger *zerolog.Logger, folderPath, key string, value string) {
 		logger.Error().Err(err).Msgf("could not store %s=%s configuration for folder %s", key, value, folderPath)
 	}
 }
+
+func GitRepoFolderPath(folderPath string) (string, error) {
```

Comment:
moved this here because of circular dependency

Reply:
The name could be a bit more expressive... Maybe `GitRepoPath`?